ui/app/services/flags.ts

```diff
@@ -31,15 +31,15 @@ export default class flagsService extends Service {
     this.flags = flags;
   }
 
-  get managedNamespaceRoot() {
-    if (this.flags && this.flags.includes(FLAGS.vaultCloudNamespace)) {
-      return 'admin';
-    }
-    return null;
+  get isManaged(): boolean {
```

Comment:
While reading the template conditionals, it wasn't super intuitive what "managed" means (because users are either self-managed or cloud managed, right?). I wonder if updating this getter to be `isCloudManaged` would be a little clearer?

Reply:
Full disclosure, I tried to follow terminology that existed before this project. So in meetings HVD clusters are talked about as "managed clusters" not "cloud managed clusters". Also, the term managed is used pre-existing code so I'm hesitant to change the terminology. Ex of code usage: the use of `managedNamespaceRoot` [here](https://github.com/hashicorp/vault/blob/main/ui/app/controllers/vault/cluster/auth.js#L25). 

Curious @noelledaley if you have any thoughts here.

Reply:
claire brings up a dilemma that has existed since the dawn of HVD 😂

i think it's important for our code to be intuitive to the broader team both now and in the future, so i'd vote for `isCloudManaged` or even `isHVDManaged`. `managedNamespaceRoot` could be renamed `cloudManagedNamespaceRoot` or `hvdManagedNamespaceRoot` (perhaps in a separate PR). 

angel, i think you're wise to follow existing terminology, but in this case since there's a small amount of code so far it's okay to change the name. 👍

Reply:
Roger that. i'll go ahead and make a follow up ticket to rename accordingly.